lib/chef/provider/aws_internet_gateway.rb

```diff
@@ -0,0 +1,78 @@
+require 'chef/provider/aws_provider'
```

Comment:
Yeah, we probably should unless otherwise notified.  Worth asking in #chef, maybe opinion has moved on :)